data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt

```diff
@@ -0,0 +1,49 @@
+package io.github.droidkaigi.feeder.data
+
+import io.github.droidkaigi.feeder.Contributor
+import io.github.droidkaigi.feeder.data.response.ContributorResponse
+import io.github.droidkaigi.feeder.data.response.ContributorsResponse
+import io.ktor.client.HttpClient
+
+open class KtorContributorApi(
+    private val authApi: AuthApi,
+    private val httpClient: HttpClient,
+) : ContributorApi {
+    override suspend fun fetch(): List<Contributor> = authApi.authenticated {
+        // TODO: 2021/03/06 remove comment out when api implemented
```

Comment:
Thank you for implementing the API even though it's not yet available! 👍 
Can you make the Fake API and change the test to work using the Fake API?
I think FeedViewModelTest will be helpful.
https://github.com/DroidKaigi/conference-app-2021/blob/f6d311de4fb522d412f5786022474305b7048ac0/android/src/test/java/io/github/droidkaigi/feeder/FeedItemViewModelTest.kt#L112-L118

Reply:
It is okay to implement it with this PR.
https://github.com/DroidKaigi/conference-app-2021/pull/299

Reply:
sorry, I understood it to create fake response.
ok! I can make it!

Reply:
Sorry 🙇

Reply:
>It is okay to implement it with this PR.

ok, I will make it on #299